src/test/java/com/hyperwallet/clientsdk/model/HyperwalletPrepaidCardTest.java

```diff
@@ -11,8 +11,8 @@ protected HyperwalletPrepaidCard createBaseModel() {
         prepaidCard
                 .token("test-token")
                 .type(HyperwalletTransferMethod.Type.PREPAID_CARD)
-
                 .status(HyperwalletTransferMethod.Status.ACTIVATED)
+                .transition(HyperwalletTransferMethod.Status.ACTIVATED)
```

Comment:
Transition status is different for different objects. Please check the document and update.